### DISCUSSION TITLE
Why could that happen that browser freezes in new WPF project

### DISCUSSION THREAD

**Discussion by lofti198** [cefsharp/CefSharp#4745] [General]
.NET Framework 4.8
CEF 121.3.130.0

Just added cefsharp in separate usercontrol inside WPF application. Somewhy it constantly freezes (for instance wher I enter text in google search form). Why could that occur?
![image](https://github.com/cefsharp/CefSharp/assets/42960488/a1bdf14e-5d19-426c-922f-e0540534984c)

**Comment by amaitland** [comment]
Check the log file see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

The render process probably crashed.
